### ISSUE TITLE
Visual Bugs with Shaders

### ISSUE THREAD

**Issue by S33R-OF-PYR3** [Wakaba2/World-Renderer#22]
I've had a couple bug reports in my mod which uses this plugin.
One person was using shaders (Slidur's Vibrant Lite & Bliss) and sometimes had a visual bug where the sky box south face was overlayed on the screen. They also tested the Solas shader but did not see the bug (labelled screenshots attached).

Another person had the shader mod Embeddium installed and reported a visual bug where the skybox was broken into more pieces and flashing various skybox textures when they moved their camera (screenshot labelled flash attached).

TLDR using shaders or just having a shader mod installed is making the sky textures behave sporadically. Addressing every unique mod and shader's quirks is a bit of an impossible order, but if you're aware of any mods or shaders that do function well with this plugin, I'd appreciate a list that I can recommend to people using my mod.

Genuinely your plugin is a life saver, so thank you for your hard work.
